plugins/flespiConnector.ts

```diff
@@ -112,24 +145,71 @@ function createClient (): MqttClient {
 /**
  * Convert the position string to an actual object
  */
-function convertToObject (json: string): Position {
+function convertToObject(json: string): Position {
     return JSON.parse(json);
 }
```

Comment:
This should not be a seperate function, just do it in-line like this:

```ts
let data: Position = JSON.parse(json);
```

Also, this doesn't seem to be used?

Reply:
This function was removed